tpch-scripts/TpchQueryExecute.sh

```diff
@@ -2,14 +2,23 @@
 #usage: TpchQueryExecute.sh SCALE_FACTOR QUERY_NUMBER
 # This script runs the hive queries on the data generated from the tpch suite and reports query execution times
 
-if [ $# -ne 2 ]
+if [ $# -lt 2 ]
 then
-	echo "Usage: ./TpchQueryExecute.sh SCALE_FACTOR QUERY_NUMBER"	
+	echo "Usage: ./TpchQueryExecute.sh SCALE_FACTOR QUERY_NUMBER [RUN_ID]"	
 	exit 1
 else
 	SCALE="$1"
 fi
 
+STARTTIME="`date +%s`"
+
+if [ -z "$3" ]
+then
+        RUN_ID=$STARTTIME
```

Comment:
Are you sure this is correct? This will generate different RUN_ID because then the earlier calls of date which is used to generate RUN_ID.